app/src/main/java/io/crate/plugin/PluginLoaderPlugin.java

```diff
@@ -98,33 +101,22 @@ public Settings additionalSettings() {
 
     @Override
     public Collection<Class<? extends LifecycleComponent>> getGuiceServiceClasses() {
-        Collection<Class<? extends LifecycleComponent>> nodeServices = new ArrayList<>();
-        nodeServices.addAll(sqlPlugin.getGuiceServiceClasses());
-        nodeServices.addAll(pluginLoader.getGuiceServiceClasses());
-        return nodeServices;
+        return Lists2.concat(sqlPlugin.getGuiceServiceClasses(), pluginLoader.getGuiceServiceClasses());
     }
 
     @Override
     public Collection<Module> createGuiceModules() {
-        Collection<Module> modules = new ArrayList<>();
-        modules.add(new PluginLoaderModule(pluginLoader));
-        modules.addAll(pluginLoader.createGuiceModules());
-        modules.addAll(sqlPlugin.createGuiceModules());
-        return modules;
+        return Lists2.concat(pluginLoader.createGuiceModules(), sqlPlugin.createGuiceModules());
```

Comment:
Is there a reason the order is opposite than here: https://github.com/crate/crate/pull/5734/files#diff-3d9437d1906f269f3e8e685a50d0d662R104